### PR TITLE
feat(landing page): create popular items and its appropriate test

### LINKED ISSUE #15: Popular Items Section
**Create Popular Items Section for Landing Page**: 

**Description**:
- **Task**: Develop a section for featuring popular items on the "Let's Share" landing page.
- **Objective**: Design and implement a section that showcases popular items to attract user engagement.

**Subtasks**:
1. **Popular Items Section Design**: Create a design concept for the popular items section.
   - *Description*: Define the layout, content, and visual elements of the section to display popular items using TailwindCSS classes.

2. **Popular Items Section Development**:  Implement the popular items section according to the design concept using Next.js components.
   - *Description*: Create Next.js components for the popular items section and use TailwindCSS classes to style the section.

3. **Responsive Design**: Ensure the popular items section is responsive for various screen sizes, including mobile devices.
   - *Description*: Apply CSS styling to make the popular items section adapt to different screen dimensions.

4. **Snapshot Testing**: Implement snapshot testing for the Popular Items Section component.
   - *Description*: Apply responsive design techniques with TailwindCSS classes to adapt the popular items section to different screen dimensions.

5. **Dummy Data Array Creation**: Create an array of 5 objects with dummy data to display in the popular items section on the home page temporarily until the database data is available.
   - *Description*: Generate sample data for the popular items section for initial development and testing purposes.

6. **ProductCard Component Creation**: Develop a ProductCard component for displaying individual product cards.
   - *Description*: Build a reusable component to represent product cards, including relevant information and styling.

7. **ProductCard Snapshot Testing**: Implement snapshot testing for the Popular Items Section component.
   - *Description*: Write snapshot tests using jest to capture and compare the rendering of the Popular Items Section component to detect unexpected changes.

8. **ProductCard Component Creation**: Develop a ProductCard component for displaying individual product cards.
   - *Description*: Create a reusable Next.js component to represent product cards, including relevant information and styling using TailwindCSS classes.

9. **ProductCard Snapshot Testing**: Implement snapshot testing for the ProductCard component.
   - *Description*: Write snapshot tests to capture and compare the rendering of the ProductCard component using jest.

10. **Data Mapping and Display**: Map over the data array and display product cards in the popular items section.
   - *Description*: Dynamically render product cards in the popular items section using the map function.

**Acceptance Criteria**:
- [x] The popular items section effectively showcases popular items.
- [x] The popular items section is functional and responsive on various screen sizes, including mobile devices.
- [x] Snapshot tests for the Popular Items Section and ProductCard component are implemented and passed successfully.
- [x] An array of dummy data is created and displayed in the popular items section.
- [x] The ProductCard component is developed and displays product information correctly.

**Definition of Done**:
- [x] All subtasks are completed.
- [x] Code changes are reviewed and approved by the team.
- [x] The popular items section on the landing page features popular items.
- [x] Snapshot tests are in place and validate the Popular Items Section and ProductCard component.
- [x] Dummy data is successfully displayed in the popular items section.
- [x] The ProductCard component is functional and correctly displays product information.

**Screen Shots**:

-Desktop:

Image

-Mobile:

Image

### PR DESCRIPTION
**Pull request by liliumorion**
fixes #15

Desktop view: 
image

Mobile view: 
image

# Related Issue

- Resolve #15

### CODE REVIEW

**Review by riadhmouamnia — APPROVED** [review]
## Approval Review for "Popular Items" PR

**Reviewer:** @riadhmouamnia 

Great job on the "Popular Items" PR! You've successfully created a section that includes reusable component cards, and comprehensive testing and is responsive to different screen sizes. 

The attention to detail and effort you've put into this section is commendable, and it aligns perfectly with our project's goals.

Wait for another review from a teammate then this PR will be ready for merging. Well done!

@riadhmouamnia

**Review by Bushra369 — APPROVED** [review]
Nice job on the "Popular Items" PR! You've done a great job creating a section with reusable cards, thorough testing, and making it work well on different screens. Your attention to detail and hard work on this section are really impressive and fit perfectly with our project's goals.

**Pull request merged**
